src/apis/axiosInstance.ts

```diff
@@ -11,13 +12,10 @@ const axiosInstance = axios.create({
 
 //axiosInterceptor
 const onRequest = (config: InternalAxiosRequestConfig) => {
-  const { method, url } = config;
-  console.log('config', config);
-  console.log(`${method} - ${url}`);
-
   const accessToken = getCookie({ name: 'accessToken' });
-  const token = accessToken ?? '쿠키를 찾을 수 없습니다.';
-  config.headers.Authorization = `Bearer ${token}`;
+  if (!config.headers.Authorization && accessToken) {
+    config.headers.Authorization = `Bearer ${accessToken}`;
+  }
```

Comment:
❓`accessToken` 여부만 있으면 되지 않을까 생각이 드는데 `!config.headers.Authorization`를 추가하신 이유가 있을까요?
```suggestion
  if (accessToken) {
    config.headers.Authorization = `Bearer ${accessToken}`;
  }
```

Reply:
오홍 그렇네요 수정했습니다